### ISSUE TITLE
Spaxel.model_flux has counter-intuitive meaning

### ISSUE THREAD

**Issue by bretthandrews** [sdss/marvin#173]
`Spaxel.model_flux` means measured flux that was modeled by the DAP. My intuition was that it was the flux values of the model spectrum.

```
cube = Cube(mangaid='1-209232')
spax = cube.getSpaxel(17, 17, xyorig='lower', modelcube=True)
(spax.model_flux.flux - spax.spectrum.flux).max()  # 4.9987411498975121e-06
```
Similarly, we should handle the binned fluxes and their model fits within `Bin` as `Bin.spectrum` and `Bin.model` with no need for a `Bin.model_flux`.

**Comment by bretthandrews** [comment]
Link to #246

**Comment by albireox** [comment]
Closed with #246
